drf_integrations/models.py

```diff
@@ -270,7 +270,7 @@ class Meta:
             on_delete=models.PROTECT,
             related_name="installations",
         )
-        config = fields.JSONField(null=True)
+        config = fields.JSONField(null=True, blank=True)
```

Comment:
I think Django will create a migration for this which is missing from the PR.

Reply:
Thank you for the reminder Gary!